packages/MSBot/docs/add-services.md

```diff
@@ -38,7 +38,7 @@ With the following options
 | -t, --tenantId <tenantId>                     | Azure Tenant id (either GUID or xxx.onmicrosoft.com)      |
 | -s, --subscriptionId <subscriptionId>         | Azure Subscription Id                                     |
 | -r, --resourceGroup <resourceGroup>           | Azure resource group name                                 |
-| -s, --serviceName <serviceName>               | Azure service name                                        |
+| -S, --serviceName <serviceName>               | Azure service name                                        |
```

Comment:
Is this a good idea? having case sensitive argument names is valid, but seems like it could be avoided. None of our other names are case sensitive. 

Perhaps "-n" or even NO shorthand and just require --serviceName